src/styles/_variables.scss

```diff
@@ -0,0 +1,5 @@
+// Re-export brand colour variables/functions for easier re-use within components
+@import "node_modules/govuk-frontend/govuk/settings/colours-applied";
+
+$aside-padding-top: 0.75em;
+$aside-border-top: 2px;
```

Comment:
If these variables only relate to the `aside` component, is there value keeping them in `Aside.module.scss` to reduce misdirection?